Annotato/Annotato/Views/Main/Auth/AuthViewController.swift

```diff
@@ -95,7 +97,12 @@ extension AuthViewController: AnnotatoAuthDelegate, AlertPresentable {
     }
 
     func logInDidSucceed() {
-        WebSocketManager.shared.setUpSocket()
+        guard let userId = AnnotatoAuth().currentUser?.uid else {
+            AnnotatoLogger.error("Unable to retrieve user id.", context: "AuthViewController::logInDidSucceed")
+            return
+        }
+
+        webSocketManager?.setUpSocket(urlString: RemotePersistenceService.generateWebSocketUrlString(userId: userId))
```

Comment:
Do we need to do this here? Considering that the `webSocketManager.setUpSocket` will be called when we replace the root view controller after logging in

Reply:
`webSocketManager.setUpSocket` will not be called in specific edge cases. For example, if the user is logging in with internet, then it won't call the `webSocketManager.setUpSocket` in `NetworkMonitor::start`